packages/phasor/src/surface.ts

```diff
@@ -32,6 +32,10 @@ export class SurfaceManager {
     this._yElements.observeDeep(this._handleYEvents);
   }
 
+  get elements() {
```

Comment:
This should not be exposed, it's concerning to let users fully traverse all elements.

Reply:
Marked as `@deprecated` ATM